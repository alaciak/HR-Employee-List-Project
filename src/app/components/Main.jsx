import React from 'react';
import Nav from '../components/Nav.jsx';

export const Main = props => {
  return (
    <div>
      <Nav/>
      { props.children }
    </div>
  );
}
