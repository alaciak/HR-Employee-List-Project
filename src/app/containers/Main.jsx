import React from 'react';
import ReactDOM from 'react-dom';
import Nav from '../components/Nav.jsx';

class Main extends React.Component {

  render() {
    return (
      <div>
        <Nav/>
        {this.props.children}
      </div>
    );
  }
}

module.exports = Main;
