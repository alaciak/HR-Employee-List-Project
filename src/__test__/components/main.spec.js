import React from 'react';
import { Main } from '../../app/components/Main.jsx';
import { shallow, render } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Main', () => {

  it('should render if Nav component with children (if they are passed)', () => {

    const wrapper = shallow(<Main />);

    expect(toJson(wrapper)).toMatchSnapshot();

  });

});
