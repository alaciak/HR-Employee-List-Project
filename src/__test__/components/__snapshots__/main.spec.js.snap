// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render Nav component 1`] = `
<div>
  <Translate(Nav) />
</div>
`;
